### ISSUE TITLE
Diff between caffe models.

### ISSUE THREAD

**Issue by logic1988** [NVlabs/PWC-Net#80]
What's the difference between four caffe models? Which model has the highest accuracy?Which one equal to the pytorch/pwc_net.pth.tar? Thank you!

**Comment by deqings** [comment]
Sorry for the confusion and late reply.

./model/pwc_net.caffemodel PWC-Net_ROB fine-tuned using Sintel, KITTI, and HD1K data; tends to perform robustly across different datasets

./model/pwc_net_chairs_things.caffemodel PWC-Net-ROB pre-trained using FlyingChairs and FlyingThings, probably lowest accruacy

./model/PWC-Net_plus/PWC-Net_plus_sintel/pwc-net-plus-sintel.caffemodel PWC-Net+ fined-tuned on Sintel, as reported in Table 1 of  this [extended journal version](https://arxiv.org/pdf/1809.05571.pdf), highest accuracy on Sintel benchmark

./model/PWC-Net_plus/PWC-Net_plus_kitti/pwc-net-plus-kitti.caffemodel PWC-Net+ fined-tuned on KITTI, as reported in this Table 3 [extended journal version](https://arxiv.org/pdf/1809.05571.pdf), highest accuracy on KITTI benchmark
